storybookjs/storybook
# CLI: Point the update-notice to the changelog in the suggested version

Issue:

The upgrade notice when starting Storybook always points to the CHANGELOG in the `next` branch. However, if e.g. you're in the 6.x releases, the `next` branch doesn't actually have the changelog for the next 6.x release. An easy way to fix this is to always point to the CHANGELOG residing in the tag corresponding to the suggested version.

## What I did

Changed `next` as a ref in the changelog URL to the ref of the suggested version

## How to test


If your answer is yes to any of these, please make sure to include it in your PR.



> This affects the current 6.x releases, so I don't know if I should submit this PR to the `main` or `next` branch.
